docs/docs/cmd/spo/user/user-get.md

```diff
@@ -44,3 +44,51 @@ Get user with login name 'i:0#.f
 ```sh
```

Comment:
Seems like this example message is not formatted well, could you also fix this one please?